prompt_toolkit/key_binding/bindings/emacs.py

```diff
@@ -438,10 +438,10 @@ def unshift_move(event: E) -> None:
 
         try:
             # Both the dict lookup and `get_by_name` can raise KeyError.
-            handler = get_by_name(key_to_command[key])
+            handler = get_by_name(key_to_command[key]).handler
         except KeyError:
             pass
-        else:  # (`else` is not really needed here.)
+        else:  # (We don't want the handler to execute if there was a KeyError)
```

Comment:
This corresponds to the comment given in this diff https://github.com/prompt-toolkit/python-prompt-toolkit/pull/989/files#r344139334